pyproject.toml

```diff
@@ -21,12 +21,6 @@ giskard = [
 ]
 
 
-[[tool.pdm.source]]
```

Comment:
Unrelated changes ? or is it because branch is not up to date ?

Reply:
not related to this PR